Create TAI-MAI.md

dileepbk opened the pull request:
Additional PA (Platform Adapter).
New MAI (Module) and its interface to TAI (Transponder) Abstraction Interface.

dileepbk committed:
Create TAI-MAI.md

First Draft

ishidawataru commented:
@dileepbk Thanks for the draft.
Can you remove the explanation of TAI because it is already described in README.md?
Let's focus on MAI in this document.

dileepbk committed:
Removed few TAI

I want to keep the TAI reference to be included in this doc which explains both TAI-MAI.
It will be not clear to user, if they have to refer to two doc and infer the relation between the two.
